### PR TITLE
bhyve/audio.c: avoid re-calculating the length of dev_name

### PR DESCRIPTION
**Pull request by rilysh**
In the function audio_init(), strlen() is being called two times, first to get the length of dev_name and second to use in memcpy(). Creating a local variable and keeping the length avoids this re-calculation.

### CODE REVIEW

File: usr.sbin/bhyve/audio.c [hunk 1]
```diff
@@ -80,15 +80,17 @@ audio_init(const char *dev_name, uint8_t dir)
 #endif
 	};
 #endif
+	size_t sz;
```
Comment (gmshake) [inline usr.sbin/bhyve/audio.c:83]: I'd prefer nlen to sz.

**Review by gmshake — APPROVED** [review]
Looks good to me.

File: usr.sbin/bhyve/audio.c (hunk 1 above)
Comment (rilysh) [reply usr.sbin/bhyve/audio.c]: Thanks! I've updated the name.

**Review by bsdimp — APPROVED** [review]
This is ready to land.

**Comment by bsdimp** [comment]
This has landed. Thank you for your contribution.

**Pull request closed without merging**